apps/web-giddh/src/app/inventory/components/stock-report-component/inventory.stockreport.component.ts

```diff
@@ -1,328 +1,330 @@
+        },
+        {
+            "value": "CREDIT_NOTE",
+            "label": "Credit Note",
+            "checked": true
+        },
+        {
+            "value": "DEBIT_NOTE",
+            "label": "Debit Note",
+            "checked": true
+        }
+    ];
+    public CategoryOptions: IOption[] = [
+        {
+            value: "dr",
+            label: "Inwards",
+            disabled: false
+        },
+        {
+            value: "cr",
+            label: "Outwards",
+            disabled: false
+        },
+        {
+            value: "dr",
+            label: "Opening Stock",
+            disabled: false
+        },
+        {
+            value: "cr",
+            label: "Closing Stock",
+            disabled: false
+        }
+    ];
+
+    public CategoryTypeOptions: IOption[] = [
+        {
+            value: "qty",
+            label: "Quantity",
+            disabled: false
+        },
+        {
+            value: "amt",
+            label: "Amount",
+            disabled: false
+        }
+    ];
+
+    public FilterValueCondition: IOption[] = [
+        {
+            value: "equal",
+            label: "Equals",
+            disabled: false
+        },
+        {
+            value: "greater_than",
+            label: "Greater than",
+            disabled: false
+        },
+        {
+            value: "less_than",
+            label: "Less than",
+            disabled: false
+        },
+        {
+            value: "not_equals",
+            label: "Excluded",
+            disabled: false
+        }
+    ];
+    public datePickerOptions: any = {
+        hideOnEsc: true,
+        locale: {
+            applyClass: 'btn-green',
+            applyLabel: 'Go',
+            fromLabel: 'From',
+            format: 'D-MMM-YY',
+            toLabel: 'To',
+            cancelLabel: 'Cancel',
+            customRangeLabel: 'Custom range'
+        },
+        ranges: {
+            'Last 1 Day': [
+                moment().subtract(1, 'days'),
+                moment()
+            ],
+            'Last 7 Days': [
+                moment().subtract(6, 'days'),
+                moment()
+            ],
+            'Last 30 Days': [
+                moment().subtract(29, 'days'),
+                moment()
+            ],
+            'Last 6 Months': [
+                moment().subtract(6, 'months'),
+                moment()
+            ],
+            'Last 1 Year': [
+                moment().subtract(12, 'months'),
+                moment()
+            ]
+        },
+        startDate: moment().subtract(30, 'days'),
+        endDate: moment()
+    };
+    public stockReport: StockReportResponse;
+    public universalDate$: Observable<any>;
+    public selectedCompany$: Observable<any>;
+    public selectedCmp: CompanyResponse;
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
     public advanceSearchModalShow: boolean = false;
+    public UpdateStockSuccess$: Observable<boolean>;
```

Comment:
Add JS doc and use proper camelCase in variable name
```suggestion
    public updateStockSuccess$: Observable<boolean>;
```